azure-pipelines.yml

```diff
@@ -450,7 +454,15 @@ stages:
             env:
               NativeToolsOnMachine: true
             displayName: Build / Test
-            condition: ne(variables['_testKind'], 'testIntegration')
+            condition: and( ne(variables['_testKind'], 'testIntegration'), ne(variables['System.JobName'], 'transparent_compiler') )
```

Comment:
AFAIU we have some flaky tests there, should we make it continue on error at least for now?

Reply:
I'm not touching what happens with integration tests. 

And the transparent compiler tests shouldn't be flaky - and if they are it needs to be resolved.

Reply:
Yeah I meant transparent compiler tests and all the recent PRs about flaky tests in AsyncMemoise and so on.

Reply:
Well those are already in the main suite... so just gotta fix them :)